README.md

```diff
@@ -10,10 +10,43 @@ The Vault Operator deploys and initializes Vault on Kubernetes, and runs in high
 
 Deploy the charm:
 ```bash
-juju deploy vault-k8s -n 5 --trust
+juju deploy vault-k8s --channel edge -n 5 --trust
 ```
 > Note: It is advised to deploy Vault with an odd number of units
 
+We recommend deploying Vault with an odd number of units.
+
+### Reach Vault through Ingress
+
+#### Deploy Traefik
+
+```bash
+juju deploy traefik-k8s --channel edge --trust --config external_hostname=<your hostname> 
```

Comment:
Any reason to use the edge channel? Are some of the tls features not available in beta or stable?

Reply:
The version 2 of the interface is currently supported properly only on edge. It breaks some setup if deployed with stable, but in some kind of half-working state which is confusing.

Reply:
The traefik in edge or beta channel uses ingress v2 but stable uses only v1. I choose edge to deploy with latest version.  However, beta channel could be preferred.  I will update it as beta.